<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sender</class>
 <widget class="QWidget" name="sender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>5</number>
     </property>
     <widget class="QWidget" name="side_send">
      <widget class="QPushButton" name="x_0_send">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="lakkering">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lakkering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="elektrokjemisk_metode">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Elektrokjemisk metode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="kjemisk_metode">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kjemisk metode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="herding">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>430</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Herding</string>
       </property>
      </widget>
      <widget class="QPushButton" name="annet">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>430</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Annet</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tilbake_0_send">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>341</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="modusogbehandling">
        <item row="0" column="0">
         <widget class="QLabel" name="modus_0">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Valgt modus: Send</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="velgbehandling">
          <property name="font">
           <font>
            <pointsize>23</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Velg overflatebehandling</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="logo_tepvr_0">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_lakkering_send">
      <widget class="QPushButton" name="tilbake_1_lakk_send">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="modus_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>291</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Send | Lakkering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="vaatlakkering">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>VÃ¥tlakkering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="offshore_lakkering">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>250</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Offshore lakkering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pulverlakkering">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>250</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pulverlakkering</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_1">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_elektrokjemisk_metode_send">
      <widget class="QPushButton" name="tilbake_2_elkj_send">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="modus_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>401</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Send | Elektrokjemisk Metode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="eloksering_alu">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Eloksering (aluminium)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="elzink">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>430</y>
         <width>661</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Elektrolytisk forsinking/elzink</string>
       </property>
      </widget>
      <widget class="QPushButton" name="hardforkromming">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Hardforkromming</string>
       </property>
      </widget>
      <widget class="QPushButton" name="kromatering">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kromatering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="elektrolytisk_fornikling">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>430</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Elektrolytisk fornikling</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_2">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_kjemisk_metode_send">
      <widget class="QPushButton" name="tilbake_3_kj_send">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="modus_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>341</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Send | Kjemisk Metode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="brunering">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Brunering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="syrevask">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Syrevask</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fosfatering">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fosfatering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="galvanisering_varmforsinking">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>430</y>
         <width>661</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Galvanisering/varmforsinking</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_3">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_herding_send">
      <widget class="QPushButton" name="tilbake_4_herd_send">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="modus_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Send | Herding</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nitrering">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nitrering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="plasma_nitrering">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>430</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Plasma nitering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sett_herding">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sett-herding</string>
       </property>
      </widget>
      <widget class="QPushButton" name="herd_herding">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Herding</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_4">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_annet_send">
      <widget class="QPushButton" name="tilbake_5_annet_send">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="modus_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Send | Annet</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sliping">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sliping</string>
       </property>
      </widget>
      <widget class="QPushButton" name="glass_blaasing">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>100</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Glass blÃ¥sing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sandblaasing">
       <property name="geometry">
        <rect>
         <x>940</x>
         <y>100</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SandblÃ¥sing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="traadskjaering">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>100</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TrÃ¥dskjÃ¦ring</string>
       </property>
      </widget>
      <widget class="QPushButton" name="polering">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>510</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Polering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="elektrolytisk_polering">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>510</y>
         <width>391</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Elektrolytisk polering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="forsolving_gullbelegging">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>300</y>
         <width>391</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ForsÃ¸lving/Gullbelegging</string>
       </property>
      </widget>
      <widget class="QPushButton" name="gullbelegging">
       <property name="geometry">
        <rect>
         <x>940</x>
         <y>300</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Gullbelegging</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fortinning">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>300</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fortinning</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_5">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
